packages/p2p/src/components/composite-calendar/calendar-side-list/calendar-side-list.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { toMoment } from '@deriv/shared';
+import ListItem from './list-item';
+
+type TItem = {
+    value: string;
+    label: string;
+    onClick: () => void;
+    duration: number;
+};
+
+type TCalendarSideListProps = {
+    from: number;
+    items: Array<TItem>;
+    to: number;
+};
+
+const isActive = (from: number, to: number, flag: number) => {
+    if (flag === 0) {
+        return toMoment().endOf('day').unix() === to && from === null;
+    }
+    return Math.ceil(to / 86400) - Math.ceil(from / 86400) === flag;
+};
```

Comment:
could you add some tsdoc description to explain what this does? 

also why use the number 86400? just curious

Reply:
86400 => number of seconds in a day.
this calculates the duration of the time range in days, hence dividing by seconds. i'll add a tsdoc for it.